operators/index.adoc

```diff
@@ -0,0 +1,41 @@
+
+An Operator extends the Kubernetes API. With Operators, you an create applications to monitor the running services in the cluster.
+Operators are designed specifically for your applications. Operators implement and automate the common Day 1 operations such as installation and configuration as well as Day 2 operations such as auto-scaling up and down and backups. All these activities are in a piece of software running inside your cluster.
+To understand more about Operators, see xref:../operators/understanding/olm-what-operators-are.adoc#olm-what-operators-are[What are Operators?]
+
+[id="developer-tasks"]
+== For developers
+
+As a developer, you can perform the following Operator tasks:
+
+** xref:../operators/user/olm-creating-apps-from-installed-operators.adoc#olm-creating-apps-from-installed-operators[Create an application from an installed Operator through the web console].
+** xref:../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Install and subscribe an Operator to your namespace].
+** xref:../operators/operator_sdk/osdk-about.adoc#osdk-about[Use Operator SDK to build,test, and deploy an Operator].
+** xref:../operators/operator_sdk/osdk-installing-cli.adoc#osdk-installing-cli[Install Operator SDK CLI].
+** Create xref:../operators/operator_sdk/golang/osdk-golang-quickstart.adoc#osdk-golang-quickstart[Go-based Operators], xref:../operators/operator_sdk/ansible/osdk-ansible-quickstart.adoc#osdk-ansible-quickstart[Ansible-based Operators], and xref:../operators/operator_sdk/helm/osdk-helm-quickstart.adoc#osdk-helm-quickstart[Helm-based Operators].
+
+[id="administrator-tasks"]
+== For administrators
+
+As a cluster administrator, you can perform the following Operator tasks:
+
+** xref:../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Install an Operator from OperatorHub]
+** xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Upgrade installed Operators]
+** xref:../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[Delete installed Operators]
+** xref:../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[Configure proxy support]
+** xref:../operators/admin/olm-status.adoc#olm-status[View Operator status].
+** xref:../operators/admin/olm-managing-operatorconditions.adoc#olm-managing-operatorconditions[Manage Operator conditions]
+** xref:../operators/admin/olm-creating-policy.adoc#olm-creating-policy[Allow non-cluster administrators to install Operators]
+** xref:../operators/admin/olm-managing-custom-catalogs.adoc#olm-managing-custom-catalogs[Manage custom catalogs]
+** xref:../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Use Operator Lifecycle Manager on restricted networks]
+
+To know all about the Operators that Red Hat provides, see xref:../operators/operator-reference.adoc#red-hat-operators[Red Hat Operators]
```

Comment:
I recently merged the following PR, which renames the "Red Hat Operators" assembly to "Platform Operators reference", and provides some more context. I might suggest something like the following change to this line here, as a result:

```
For an index of CVO-managed Operators, see xref:../operators/operator-reference.adoc#platform-operators-ref[Platform Operators reference].
```

Reply:
Expanded CVO  and added the above sentence.